presto-orc/src/main/java/com/facebook/presto/orc/reader/ColumnReader.java

```diff
@@ -39,7 +38,6 @@
 
     QualifyingSet inputQualifyingSet;
     QualifyingSet outputQualifyingSet;
-    Block block;
     int outputChannel = -1;
```

Comment:
Should we introduce a constant for -1 `CHANNEL_NOT_PRESENT`?

Reply:
@tdcmeehan `if (outputChannel != CHANNEL_NOT_PRESENT)` is quite verbose. How about a boolean flag: `private boolean outputChannelSet;` + '`if (!outputChannelSet)`?

Reply:
Sounds good